x/stakeibc/keeper/hooks.go

```diff
@@ -262,8 +261,7 @@ func (k Keeper) TransferExistingDepositsToHostZones(ctx sdk.Context, epochNumber
 	var emptyRecords []uint64
 	for _, depositRecord := range transferDepositRecords {
 		if depositRecord.DepositEpochNumber < cast.ToUint64(epochNumber) {
-			pstr := fmt.Sprintf("\t[TransferExistingDepositsToHostZones] Processing deposits {%d} {%s} {%d}", depositRecord.Id, depositRecord.Denom, depositRecord.Amount)
-			k.Logger(ctx).Info(pstr)
+			k.Logger(ctx).Info(fmt.Sprintf("\t[TransferExistingDepositsToHostZones] Processing deposits {%d} {%s} {%d}", depositRecord.Id, depositRecord.Denom, depositRecord.Amount))
```

Comment:
nit: The brackets here are a little confusing, might be a little cleaner to do
```
"\t[TransferExistingDepositsToHostZones] Processing deposits ID: %d, Amount: %d%s", depositRecord.Id, depositRecord.Amount, depositRecord.Denom)
```

Reply:
nice! done